net-dns/dnrd/files/dnrd

```diff
@@ -11,13 +11,13 @@ depend() {
 
 start() {
 	ebegin "Starting dnrd"
-	/usr/sbin/dnrd $DNRD_OPTS &> /dev/null &
+	/usr/sbin/dnrd $DNRD_OPTS >/dev/null 2>&1
```

Comment:
You're losing `&`, so I'm going to guess it's going to block forever? Also use start-stop-daemon maybe.